cmd/kubeadm/app/master/manifests.go

```diff
@@ -428,6 +429,11 @@ func getControllerManagerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted
 		"use-service-account-credentials":  "true",
 		"controllers":                      "*,bootstrapsigner,tokencleaner",
 	}
+	if k8sVersion.LessThan(kubeadmconstants.MinimumCSRSARApproverVersion) {
+		// enable the former CSR group approver for v1.6 clusters.
+		// TODO(luxas): Remove this once we're targeting v1.8 at HEAD
+		defaultArguments["insecure-experimental-approve-all-kubelet-csrs-for-group"] = bootstrapapi.BootstrapGroup
```

Comment:
@liggitt It was being set unconditionally in the `defaultArguments` slice initializer above. Now only for v1.6